src/server/service/kv.rs

```diff
@@ -2373,6 +2399,44 @@ fn needs_reject_raft_append(reject_messages_on_memory_ratio: f64) -> bool {
     false
 }
 
+struct HealthFeedbackAttacher {
+    store_id: u64,
+    health_controller: HealthController,
+    last_feedback_time: Option<Instant>,
+    seq: Arc<AtomicU64>,
+}
+
+impl HealthFeedbackAttacher {
+    fn new(store_id: u64, health_controller: HealthController, seq: Arc<AtomicU64>) -> Self {
+        Self {
+            store_id,
+            health_controller,
+            last_feedback_time: None,
+            seq,
+        }
+    }
+
+    fn attach_if_needed(&mut self, resp: &mut BatchCommandsResponse) {
+        let now = Instant::now_coarse();
+
+        if let Some(last_feedback_time) = self.last_feedback_time
+            && now - last_feedback_time < Duration::from_secs(1)
```

Comment:
Is it necessary to make the `Duration::from_secs(1)` dynamically configurable?

Reply:
And one more tip: how about use `last_feedback_time.elapsed()` ?